requirements/run.in

```diff
@@ -1,21 +1,21 @@
 # Since the install procedure install the packages below from pypi, we cannot
 # pin versions outside pypi, i.e. latest git master branch version.
-ipython==8.1.1
+ipython==8.22.1
```

Comment:
Can you also verify in the shell if tab auto completion is still working? Back then I hit https://github.com/kytos-ng/kytos/issues/350, but looks like the issue upstream has been resolved, so it should be working, but good to double check.

Reply:
The tab autocomplete is working. The arrows are not although no exception is raised.

Reply:
Right. Good to know that tab auto completion still works, this is probably one of the most helpful things of the embedded shell. 

The arrows up/down for history is good to have but not a massive deal breaker, I'll encourage you to check out ipython changelog and/or source code to see if anything else needs to be set to have it. 

Finally, try out `ctrl+f` too to see if it's working (if the history isn't the it might not too), but with `ctrl+f` if it detects that a prior command is the same, if you hit `ctrl+f` it inserts the command entirely, for instance, if were to hit `ctrl+f` it would insert `controller.napps[('kytos', 'mef_eline')]`, this is not a deal breaker either, but if we can manage to keep it still working, that's great:

![20240227_135552](https://github.com/kytos-ng/kytos/assets/1010796/78721d88-ec55-4095-a93d-df287b4235d3)

Reply:
Fixed

Reply:
Excellent